src/sst/core/testElements/coreTest_Serialization.cc

```diff
@@ -0,0 +1,224 @@
+// Copyright 2009-2022 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2022, NTESS
+// All rights reserved.
+//
+// Portions are copyright of other developers:
+// See the file CONTRIBUTORS.TXT in the top level directory
+// the distribution for more information.
```

Comment:
Fix to be `// of the distribution`

Reply:
Wish I could remember where I copied that from since it's likely wrong there as well.

Reply:
I've found the typo in multiple coreTest_* files.  Do we want to change them all as part of the PR or go back and change them all as a separate PR?

Reply:
Will hold off on this change to another PR where we clean up all the copyright statements.